src/main/java/mx/ori/pronouns/PronounsScreen.java

```diff
@@ -101,7 +124,7 @@ protected boolean isFocused() {
 
 
 
-        public class PronounsEntry extends Entry<PronounsEntry> {
+        public class PronounsEntry extends AlwaysSelectedEntryListWidget.Entry<PronounsEntry> {
```

Comment:
I'm not sure why my ide wasn't accepting this as it was :/ in any case it's more explicit, at least.